bin/main.rb

```diff
@@ -0,0 +1,48 @@
+
+require_relative '../lib/game'
+require_relative '../lib/board'
+require_relative '../lib/player'
```

Comment:
I noticed this class is being required but I don't see it initialized anywhere, are you using yet?